src/shared/components/login/login.js

```diff
@@ -56,15 +56,20 @@ class Login extends Component {
   }
 
   render() {
-    const { error } = this.state;
+    let { error } = this.state;
+    if (error === 'Request failed with status code 401') {
```

Comment:
Thanks! 

Not sure how I missed it before, but just noticed that the FormInput and FormEmail components on this page are causing React warnings (their id props aren't being set but they're required in prop-types). I'm going to make two commits and fix that while I'm here.